src/pages/AddRecord/AddRecordCategory.tsx

```diff
@@ -40,11 +49,47 @@ function AddRecordCategory({
       { title: '내편이 되어주세요', choosed: false, id: 10, iconSrc: MySide },
     ],
   }
-
   const [categoryState, setCategoryState] = useState<CategoryType>(categoryData)
   const [formData, setFormData] = useRecoilState(formDataAtom)
 
+  useEffect(() => {
+    const getData = async () => {
+      const { data } = await getCategory()
+      setCategoryState(makeCategoryData(data))
+      console.log(data)
+    }
+    getData()
+  }, [])
+
+  const makeCategoryData = (data: CategoryDatas) => {
+    const categoryData: CategoryType = {
+      [TEXT_DETAILS.CELEBRATION]: data[1].subcategories.map(
+        (category: BigCategory, index: number) => {
+          return {
+            title: category.name,
+            choosed: index === 0 && true,
+            id: category.id,
+            iconSrc: getIconSrc(category.id),
+          }
+        }
+      ),
+      [TEXT_DETAILS.CONSOLATION]: data[0].subcategories.map(
+        (category: BigCategory, index: number) => {
+          return {
+            title: category.name,
+            choosed: index === 0 && true,
+            id: category.id,
+            iconSrc: getIconSrc(category.id),
+          }
+        }
+      ),
+    }
+    return categoryData
+  }
```

Comment:
이부분 맵중첩으로 하는게 낫지 않아요? data랑 index가 안맞으면 상수를 바꾸면 될듯요?
```suggestion
    const categoryData: CategoryType = {
      TEXT_DETAILS.map((item, index) => 
        [item]: data[index].subcategories.map(...
        ))
      ...
    return categoryData
  }
```